src/modelfree/policies/transparent.py

```diff
@@ -33,6 +33,7 @@ def _get_default_transparency_dict(self, obs, ff, hid):
         :param ff: (dict<str>, [float]) dictionary of lists of feedforward activations.
         :param hid: ([float] or None) LSTM hidden state.
         """
+        # TODO: Do not consolidate -- have this happen later down the pipeline.
```

Comment:
Did you want to do this in this PR, or just leave this as a note for a thing to do sometime in the future?

Reply:
Intended as a note to future. It's not breaking anything right now, it's just not the most flexible design.